tests/v2/test_waku_rln_relay.nim

```diff
@@ -136,125 +136,126 @@ proc uploadContract(ethClientAddress: string): Future[Address] {.async.} =
   return contractAddress
 
 procSuite "Waku rln relay":
-  asyncTest  "contract membership":
-    let contractAddress = await uploadContract(EthClient)
-    # connect to the eth client
-    let web3 = await newWeb3(EthClient)
-    debug "web3 connected to", EthClient
-
-    # fetch the list of registered accounts
-    let accounts = await web3.provider.eth_accounts()
-    web3.defaultAccount = accounts[1]
-    let add = web3.defaultAccount 
-    debug "contract deployer account address ", add
-
-    # prepare a contract sender to interact with it
-    var sender = web3.contractSender(MembershipContract, contractAddress) # creates a Sender object with a web3 field and contract address of type Address
-
-    # send takes three parameters, c: ContractCallBase, value = 0.u256, gas = 3000000'u64 gasPrice = 0 
-    # should use send proc for the contract functions that update the state of the contract
-    let tx = await sender.register(20.u256).send(value = MembershipFee)
-    debug "The hash of registration tx: ", tx # value is the membership fee
-
-    # var members: array[2, uint256] = [20.u256, 21.u256]
-    # debug "This is the batch registration result ", await sender.registerBatch(members).send(value = (members.len * membershipFee)) # value is the membership fee
-
-    # balance = await web3.provider.eth_getBalance(web3.defaultAccount , "latest")
-    # debug "Balance after registration: ", balance
-
-    await web3.close()
-    debug "disconnected from", EthClient
-
-  asyncTest "registration procedure":
-    # deploy the contract
-    let contractAddress = await uploadContract(EthClient)
-
-    # prepare rln-relay peer inputs
-    let 
-      web3 = await newWeb3(EthClient)
-      accounts = await web3.provider.eth_accounts()
-      # choose one of the existing accounts for the rln-relay peer  
-      ethAccountAddress = accounts[9]
-    await web3.close()
-
-    # create an RLN instance
-    var rlnInstance = createRLNInstance()
-    check: rlnInstance.isOk == true
-
-    # generate the membership keys
-    let membershipKeyPair = membershipKeyGen(rlnInstance.value)
+#   asyncTest  "contract membership":
```

Comment:
Is this commented out on purpose? If so we should add a comment explaining why it is commented out, perhaps with a pointer to an issue describing the problem in more detail.

Reply:
Tests that need ganache-cli are now moved under the `onchain_rln` compiler flag. Please see the updated file.